spec/models/custom_button_spec.rb

```diff
@@ -294,9 +294,9 @@
   end
 
   describe "publish custom button event" do
-    let(:vm)              { FactoryGirl.create(:vm_vmware) }
-    let(:vm2)             { FactoryGirl.create(:vm_vmware) }
-    let(:vm3)             { FactoryGirl.create(:vm_vmware) }
+    let!(:vm)             { FactoryGirl.create(:vm_vmware) }
+    let!(:vm2)            { FactoryGirl.create(:vm_vmware) }
+    let!(:vm3)            { FactoryGirl.create(:vm_vmware) }
```

Comment:
Why the change to `let!`.  Variables `vm2` and `vm3` are only used in one test but will not get created for each.

Reply:
@bdunne, is ```allow(Vm).to receive(all) ...``` better than let! here? Or ... something else?

Reply:
Are they needed for the rest of the tests?  If not, https://github.com/ManageIQ/manageiq/pull/18056/files#diff-71376e22701f18c6f0be76915e41ef84R331 should create them, so you shouldn't need `let!` or the `allow(Vm).to receive(:all)...`.

Reply:
To be clear, I did this change for https://github.com/ManageIQ/manageiq/pull/18056/files#diff-71376e22701f18c6f0be76915e41ef84R349.